resources/js/Organisms/Form.tsx

```diff
@@ -81,6 +85,15 @@ const Form: React.FC<IFormProps> = ({
         }
     }, [values]);
 
+    useEffect(() => {
+        return () => {
+            // ComponentWillUnmount in Class Component
+            _isMounted.current = false;
+        };
+    }, []);
```

Comment:
This seems like the sort of thing that we might need elsewhere too in the future. If we do, then it might be worth making a custom shared hook for this - something like this:
```
const useIsMountedRef = () => {
    const isMountedRef = useRef(null);
    useEffect(() => {
        isMountedRef.current = true;
        return () => (isMountedRef.current = false);
    });
    return isMountedRef;
};
```

But for now I think it's fine to leave it in this component, seeing as this is the only component that needs it at the moment.

Reply:
ooh thanks - I've so far only written one custom hook so far and don't remember how it works!